.erb-lint.yml

```diff
@@ -39,6 +39,6 @@ linters:
       AllCops:
         DisabledByDefault: true
       Primer/DeprecatedComponents:
-        Enabled: true
+        Enabled: false
```

Comment:
Curious about this change

Reply:
this is already run in rubocop, so it's just duplicated here and didn't seem necessary. the PR description shows the duplicate output when this was running vs non-duplicate output after i disabled it